website/content/partials/datasource/hcp-packer-image/Config-not-required.mdx

```diff
@@ -0,0 +1,6 @@
+<!-- Code generated from the comments of the Config struct in datasource/hcp-packer-image/data.go; DO NOT EDIT MANUALLY -->
+
+- `component_type` (string) - The specific Packer builder or post-processor used to create the image.
```

Comment:
This is out-of-date compared to your latest version of the code, you can fix it by running `go generate` on the data.go file

Reply:
updated, thanks